src/test/java/sirius/kernel/nls/NLSSpec.groovy

```diff
@@ -168,6 +168,16 @@ class NLSSpec extends BaseSpecification {
         "0.1" | new BigDecimal("0.1")
     }
 
+    def "parseMachineString works for Amount"() {
+        expect:
+        NLS.parseMachineString(Amount.class, input) == output
+
+        where:
+        input | output
+        "0.1" | Amount.ONE.divideBy(Amount.TEN)
```

Comment:
Add a test for empty String and null which are way more intresting then these basic numbers.

Reply:
done